src/napari_threedee/mouse_callbacks.py

```diff
@@ -32,7 +32,7 @@ def add_point_on_plane(
     cursor_position_3d = np.asarray(event.position)[displayed_dims]
     intersection_3d = plane_layer.plane.intersect_with_line(
         line_position=cursor_position_3d,
-        line_direction=event._view_direction[displayed_dims]
+        line_direction=event.view_direction[displayed_dims]
```

Comment:
Due to this change, I think the mouse event tests need to be updated so that the `DummyEvent` has a `view_direction` property.

From the CI:

```python
  __________________________ test_add_point_on_plane_4d __________________________
  
  viewer_with_plane_and_points_4d = Viewer(axes=Axes(visible=False, labels=True, colored=True, dashed=False, arrows=True), camera=Camera(center=(13.5, 13.... 0x7fdd0c68c8b0>, 'Control-Shift-A': <function InteractionBoxMouseBindings._transform_active_layer at 0x7fdd0c68c820>})
  
      def test_add_point_on_plane_4d(viewer_with_plane_and_points_4d):
          viewer = viewer_with_plane_and_points_4d
          points_layer = viewer.layers['Points']
          plane_layer = viewer.layers['blobs_4d']
      
          class DummyMouseEvent:
              position = (0, 14, 14, 14)
              _view_direction = np.array((0, 1, 0, 0))
              modifiers = ['Alt']
```